specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-01-23-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,2774 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionHostNameParameter"
+          },
+          {
+            "$ref": "#/parameters/UserSessionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/sessionHosts/{sessionHostName}/userSessions/{userSessionId}/sendMessage": {
+      "post": {
+        "tags": [
+          "UserSession"
+        ],
+        "operationId": "UserSessions_SendMessage",
+        "description": "Send a message to a user.",
+        "x-ms-examples": {
+          "UserSession_SendMessage_Post": {
+            "$ref": "./examples/UserSession_SendMessage_Post.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionHostNameParameter"
+          },
+          {
+            "$ref": "#/parameters/UserSessionIdParameter"
+          },
+          {
+            "name": "sendMessageIn",
+            "description": "Object containing message includes title and message body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/SendMessageIn"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {

```

Comment:
Please use the standard error response, for example: https://github.com/Azure/azure-rest-api-specs/blob/7e9c8a45c28ada7d4a79b14134cb61e6afbcac95/specification/resources/resource-manager/Microsoft.Resources/stable/2019-07-01/resources.json#L4205

Reply:
Generally, errors shoudl also be called 'CloudError'